build.gradle.kts

```diff
@@ -14,6 +14,15 @@ plugins {
     alias(libs.plugins.sonarqube)
 }
 
+buildscript {
+    repositories {
```

Comment:
We may not need this here as the repositories are already defined in settings.gradle